fe/fe-core/src/main/java/com/starrocks/metric/PrometheusMetricVisitor.java

```diff
@@ -155,8 +158,11 @@ public void visitJvm(JvmStats jvmStats) {
     public void visit(@SuppressWarnings("rawtypes") Metric metric) {
         // title
         final String fullName = prefix + "_" + metric.getName();
-        sb.append(HELP).append(fullName).append(" ").append(metric.getDescription()).append("\n");
-        sb.append(TYPE).append(fullName).append(" ").append(metric.getType().name().toLowerCase()).append("\n");
+        if (!metricNames.contains(fullName)) {
```

Comment:
Would better add a comment for why `metricNames` need to de-duplication

Reply:
I'll add a comment.